omero/sysadmins/config.txt

```diff
@@ -1180,13 +1180,23 @@ Default: `pixelDataEventLogQueue`
 omero.pixeldata.max_plane_height
 ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
 
+With :property:`omero.pixeldata.max_plane_width` specifies
+the plane size cutoff above which a pixel pyramid should
+either be read directly from the file format or generated
+by the pixeldata service.
```

Comment:
So for planes smaller than this size will it never read pyramids from the file format?